package/contents/ui/components/graph/DiskGraph.qml

```diff
@@ -27,17 +27,17 @@ RMBaseGraph.TwoSensorsGraph {
 
     // Graph options
     realUplimits: [uplimits[0] * unit.multiplier, uplimits[1] * unit.multiplier]
-    sensorsModel.sensors: sensorsType[0] ? [`disk/${device}/read`, `disk/${device}/write`] : [`disk/${device}/write`, `disk/${device}/read`]
+    sensorsModel.sensors: sensorsType[0] ? [`disk/${device}/write`, `disk/${device}/read`] : [`disk/${device}/read`, `disk/${device}/write`]
 
-    // Override methods, for handle icons
+    // Override methods to handle icons
     _formatValue: (index, value) => {
         // Show icons
         let icon = ""
         if (root.icons) {
             if (index === readIndex) {
                 icon = " " + i18nc("Disk graph icon : Read", " R")
             } else if (index == writeIndex) {
-                icon = " " + i18nc("Disk graph icon : Write", "W")
+                icon = " " + i18nc("Disk graph icon : Write", " W")
```

Comment:
The `"W"` should not be changed to `" W"`, because both `" R"` and `"W"` are i18n keys:
https://github.com/orblazer/plasma-applet-resources-monitor/blob/47cd7e21b9727f756aa25df9cc416ed0571e3c4b/package/translate/de.po#L97-L105

If you directly changed `"W"` to `" W"`, then the i18n function for the "Disk graph icon : Write" would not work, as no i18n string can be matched with key `" W"`.

PS: the difference between `" R"`(has space) and `"W"`(no space) is intentional, to keep align of the numbers on the graph. 
![屏幕截图_20241126_193053](https://github.com/user-attachments/assets/bd51f812-c134-4cd2-a25c-cba854c5e6d4)